common/src/View/MapDocument.cpp

```diff
@@ -622,6 +644,32 @@ namespace TrenchBroom {
             }
             kdl::vec_clear_and_delete(nodesToDelete);
 
+            // Clean up persistent IDs of any groups being pasted
+            auto persistentGroupIds = kdl::create_set(allPersistentGroupIds(*m_world.get()));
+            for (auto& pair : nodesToAdd) {
```

Comment:
for (auto& [target, nodesToAddToTarget] : nodes)

Reply:
We're not using `target` in the loop, so to avoid `unused(target)` I did this.

Reply:
Hm.. IIRC that spurious warning was a gcc7 only bug. Obviously not a big deal.

Reply:
Hmm, could be, I feel like I got this warning on clang, but maybe not. I can try it and see if CI passes.

Reply:
Changed here: 3b31997b588444aa1685a48271131f97955ced50

Reply:
Fails as expected on GCC7, so I'll revert it.

Reply:
e07990292

Reply:
Rebased on master and adopted the structured bindings.